plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -375,15 +375,15 @@ public SemiTransactionalHiveMetastore getMetastore()
     public List<String> listSchemaNames(ConnectorSession session)
     {
         return metastore.getAllDatabases().stream()
-                .filter(HiveMetadata::filterSchema)
+                .filter(schemaName -> !HiveUtil.isHiveSystemSchema(schemaName))
```

Comment:
nit: `filter(not(HiveUtil::isHiveSystemSchema))`

Reply:
@phd3 sorry missed that, didn't refresh the page